src/pages/ScholarshipForm.jsx

```diff
@@ -41,7 +43,11 @@ function ScholarshipForm() {
     validationSchema,
     onSubmit: (values, { setSubmitting, resetForm }) => {
       setSubmitting(true);
-      Scholarships.new(values)
+      Scholarships.new({
+        ...values,
+        authorId: user?.uid,
+        authorEmail: user?.email,
```

Comment:
I wonder if this logic should be part of `Scholarships.new` or `Collections.new`. We'll probably end up moving it at some point when a similar pattern occurs.